cobc/cobc.c

```diff
@@ -3001,6 +3019,80 @@ remove_trailing_slash (char *data)
 	}
 }
 
+static int
+string_is_dash (const char *s)
+{
+	return (strcmp (s, COB_DASH) == 0);
+}
```

Comment:
we should request inlining here, but ... I'm not sure if we maybe should revert these parts doing the inlinig ourselves (as it was done before)

Reply:
I like the idea of a descriptive name such as `string_is_dash` for this (plus it's somewhat "easier" to write).
Would a macro be acceptable for this ? Otherwise I'll just reverse this change.